gutenberg.php

```diff
@@ -72,3 +72,9 @@ function gutenberg_pre_init() {
 
 	require_once __DIR__ . '/lib/load.php';
 }
+
+add_action( 'admin_enqueue_scripts', function () {
```

Comment:
I would recommend instead of using a closure to add a function like `gutenberg_enqueue_admin_view_transitions()`. That way it's easy to disable / customize if there are specific needs.